### DISCUSSION TITLE
Latex theorem blocks

### DISCUSSION THREAD

**Discussion by BinaryQuantumSoul** [slidevjs/slidev#1985] [Ideas]
Would be nice to have numbered named latex blocks with background similar to amsthm blocks (see [quarto](https://quarto.org/docs/authoring/cross-references.html#theorems-and-proofs) for example)

**Comment by cben** [comment]
Quote syntax `> ` makes a "box", but with default them fonts & layout get more cramped than surrounding text, which is the opposite of drawing attention we want when presenting a theorem...

Reasonable approx I found:
```markdown

<div style="background: #eee; padding: 1ex; margin: 1ex; border-radius: 2ex">

<h3 style="border-bottom: 1px solid #888">Theorem 11</h3>

... theorem text here ...

</div>

```
(blank lines necessary around block-level tags, both in slidev & for CommonMark inter-operability)  

<img width="1116" height="637" alt="bildo" src="https://github.com/user-attachments/assets/3576aa51-ebb8-4e39-aa65-00a2603be7e6" />

Can fine-tune more...  Change `#eee` for bg color.

~~Horizontal margin/padding refuse to work for some reason, even with `!important`, which looks a bit weird.~~ EDIT: no they work fine, if I set them large enough to be noticable.

**Comment by cben** [comment]
https://github.com/MuTsunTsai/slidev-theme-frankfurt is directly inspired by Beamer, and provides a beamer-like component for boxing theorems etc.
